packages/cli/src/commands/doctor/healthchecks/androidSDK.js

```diff
@@ -2,15 +2,60 @@ import chalk from 'chalk';
 import {logManualInstallation} from './common';
 import versionRanges from '../versionRanges';
 import {doesSoftwareNeedToBeFixed} from '../checkInstallation';
+import {exec} from 'child_process';
```

Comment:
Can we use `execa` for spawning process? It has a built-in promise interface and seems to work better with Windows by default

Reply:
You got it. Updated to use execa.